### PR TITLE
docs: Reduce typo count

### PR DESCRIPTION
**Pull request by waywardmonkeys**
Reduce the number of typos within the documentation.

### CODE REVIEW

File: docs/design/autodiff.md [hunk 1]
```diff
@@ -260,7 +260,7 @@ intermediaries which don't have concrete variables. When we add a pair (A, C) an
 
  ```C
 
-void f_SSA_Rev(inout DP<float> dpa, inout DP<float> dpb, float dout)
+void f_SSA_Rev(inout DP<float> dpa, inout DP<float> dpb, float doubt)
```
Comment (jkwak-work) [inline docs/design/autodiff.md:263]: I think this is `dout` correct

File: docs/design/capabilities.md [hunk 2]
```diff
@@ -129,10 +129,10 @@ For a given function definition `F`, the front end will scan its body and see wh
...
 
-If `F` *does* have one or more `[availabelFor(...)]` clauses that amount to a declared capability `C` (again in disjunctive normal form), then we can check that `C` implies `R` and error out if it is not the case.
+If `F` *does* have one or more `[availableFor(...)]` clauses that amount to a declared capability `C` (again in disjunctive normal form), then we can check that `C` implies `R` and error out if it is not the case.
 A reasonable implementation would track which calls introduced which requirements, and be able to explain *why* `C` does not capture the stated requirements.
 
-For a shader entry point, we should check it as if it had an `[availableFor(...)]` that is the OR of all the specified target profiles (e.g., `sm_5_0 | glsl_450 | ...`) ANDed with the specified stage (e.g., `fragment`).
+For a shader entry point, we should check it as if it had an `[availableFor(...)]` that is the OR of all the specified target profiles (e.g., `sm_5_0 | glsl_450 | ...`) ANDead with the specified stage (e.g., `fragment`).
```
Comment (jkwak-work) [inline docs/design/capabilities.md:135]: ANDed is correct. Maybe we can do AND-ed

**Review by jkwak-work — CHANGES_REQUESTED** [review]
I found a few mistakes but the rest looks good.
Toc.html needs to be generated by running a script. But this looks fine for now.

File: docs/design/autodiff.md (hunk 1 above)
Comment (waywardmonkeys) [reply docs/design/autodiff.md]: I didn't mean to include that change, thanks for catching it. Fixed.

File: docs/design/capabilities.md (hunk 2 above)
Comment (waywardmonkeys) [reply docs/design/capabilities.md]: This one also was unintentional. Fixed.

**Comment by waywardmonkeys** [comment]
@jkwak-work Updated per your corrections.

**Review by jkwak-work — APPROVED** [review]

**Review by expipiplus1 — APPROVED** [review]

**Comment by expipiplus1** [comment]
Great work, thanks!

**Pull request merged**